components/suggest/src/store.rs

```diff
@@ -1945,14 +1945,34 @@ mod tests {
                 "#]],
             ),
             (
-                "keyword = `soft`; AMP, Wikipedia, and AMO",
+                "keyword = `masking e`; AMO only",
```

Comment:
Just for completeness, could you please add another test for one of the single-word keywords, like `relay` or `spam`?